pkg/oauthserver/informer_test.go

```diff
@@ -0,0 +1,64 @@
+package oauthserver_test
```

Comment:
nope

Reply:
I really don't understand why we should be attempting to mock informers in our code base

Reply:
Do you have a better solution? Is there some solution already in place?

I need to mock it to unit test my code within WithOAuth.

Reply:
solution: initialize the informer with a proper indexer